components/places/src/api/matcher.rs

```diff
@@ -311,95 +309,95 @@ impl<'query, 'conn> OriginOrUrl<'query, 'conn> {
     }
 }
 
+const URL_SQL: &'static str = "
```

Comment:
I split these out to help me see what was giving me a syntax error, but I think its clearer with them not inline.